crates/miden-lib/asm/kernels/transaction/lib/prologue.masm

```diff
@@ -852,7 +852,7 @@ end
 #!   Advice stack: [
 #!      SERIAL_NUMBER,
 #!      SCRIPT_ROOT,
-#!      INPUTS_COMMITMENT,
+#!      STORAGE_COMMITMENT,
 #!      ASSETS_COMMITMENT,
 #!      ARGS,
 #!      NOTE_METADATA,
```

Comment:
The advice stack comment here looks outdated. We should have
```
...
NOTE_METADATA,
storage_len,
assets_count,
...
```

according to:

https://github.com/0xMiden/miden-base/blob/8592b0367c6ddb57a65b0773e348c3ea35c7bf0a/crates/miden-lib/src/transaction/inputs.rs#L356-358